LICENSE.txt

```diff
@@ -204,7 +186,7 @@ solely pursuant to the terms of the relevant commercial agreement.
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2013-2016 Crate.IO GmbH ("Crate")
```

Comment:
we must not modify the text of the Apache license itself

this placeholder text is a part of the license text proper, and serves as an example to downstream users of the license, instructing them on how to apply the license to their own works

Reply:
You are welcome to modify this part of the Apache license. There should not be any `downstream users of the licenses`. They will presumably fetch fresh copies of the Apache license when they start new projects. GitHub makes the Apache license one of the default options when creating a new repo.